main/ui/src/main/java/org/cryptomator/ui/common/FxmlFile.java

```diff
@@ -12,7 +12,7 @@
 	ERROR("/fxml/error.fxml"), //
 	FORGET_PASSWORD("/fxml/forget_password.fxml"), //
 	HEALTH_START("/fxml/health_start.fxml"), //
-	HEALTH_CHECK("/fxml/health_check_list.fxml"), //
+	HEALTH_CHECK("/fxml/health_check.fxml"), //
```

Comment:
"list" was added deliberately, as we decided to implement this window similar to "vault list + vault details"

Reply:
The layout is similar, but the internal structure is different. For the main window, we have the main window controller containing/including a controller for the list and a controller for the detail.

Here the list does not have its own controller. It is "only"  one element inside the bigger container.

Reply:
I don't believe that the name should not depend on the root element of the fxml file. Even though the actual "list" is contained within some other container, the sole purpose of this fxml + controller is selecting and running the list of available checks. Hence the name. As said before, it was a deliberate choice to move away from the singular `health_check`.

Reply:
Changes reverted in 66a8dc9